octoprint_mrbeam/static/js/convert.js

```diff
@@ -655,8 +655,9 @@ $(function(){
 		};
 
 		self.thickness_text = function(data){
-			if(data.thicknessMM < 0) return "engrave only";
-			else return data.thicknessMM+' mm';
+			if(data.thicknessMM < 0) return gettext("engrave only");
+			// Translators: "millimeters"
```

Comment:
Unfortunately, it does not work in JavaScript. Because OctoPrint is using such babel.js library which has only gettext and ngettext functions. However, we need pgettext to make Translation notes.